### PR TITLE
Create First-one-to-pass-truth-test.md

### PR DESCRIPTION
**Pull request by meetzaveri**
- As Angelos said, if there is native js function that solves this problem then it is not useful.

### CODE REVIEW

**Comment by Chalarangelo** [comment]
Made some minor changes, I'll check around a little bit to see if this can be improved further or if there's a native function, then merge. Thanks for contributing!

**Comment by meetzaveri** [comment]
Okay sure @Chalarangelo

**Comment by meetzaveri** [comment]
@Chalarangelo This second snippet different from first one and is based on object.

File: snippets/Truth-check-objects.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+Check if the predicate (second argument) is truthy on all elements of a collection (first argument).
+
+
+ - For every object in the collection array, check the truthiness of object’s property passed in pre parameter
+ - Array#every method internally checks if the value returned from the callback is truthy.
+ - Return true if it passes for every object. Otherwise, return false.
+ - Also if the object is empty then it will return false
+ 
+ ```
+ function truthCheck(collection, pre) {
```
Comment (Chalarangelo) [inline snippets/Truth-check-objects.md]: Mark the code block as `js` and format the code to ES6, please!

**Comment by meetzaveri** [comment]
Here I have two snippets, so are you talking about first one ? @Chalarangelo

**Comment by Chalarangelo** [comment]
Yes, the second one is already a native function.

**Comment by meetzaveri** [comment]
Then it's your choice @Chalarangelo

**Pull request merged**